docs/getting-started/perses/dashboards/cortex-writes.yaml

```diff
@@ -0,0 +1,910 @@
+kind: Dashboard
```

Comment:
We could also make the dashboards in go or cue. I don't want to do anything fancy yet. I think we have to wait and see how the community reacts. If more people are interested